$MirOS: ports/www/firesomething/patches/patch-browser_app_mozilla_in,v 1.3 2008/05/21 06:31:03 bsiegert Exp $
$OpenBSD: patch-browser_app_mozilla_in,v 1.4 2005/10/13 04:32:41 bernd Exp $
--- browser/app/mozilla.in.orig	2007-10-05 09:29:26.000000000 +0200
+++ browser/app/mozilla.in	2008-08-24 15:31:41.000000000 +0200
@@ -53,6 +53,7 @@
 #set -x
 
 moz_libdir=%MOZAPPDIR%
+MOZ_PLUGIN_PATH=_LOCALBASE_/lib/mozilla-plugins
 
 # Use run-mozilla.sh in the current dir if it exists
 # If not, then start resolving symlinks until we find run-mozilla.sh
@@ -90,6 +91,7 @@ if [ $found = 0 ]; then
   # Check default compile-time libdir
   if [ -x "$moz_libdir/run-mozilla.sh" ]; then
     dist_bin="$moz_libdir"
+    run_moz="$dist_bin/run-mozilla.sh"
   else 
     echo "Cannot find mozilla runtime directory. Exiting."
     exit 1
